### PR TITLE
feat: add trivy config file path option

### PR DESCRIPTION
**Pull request by kynefuk**
- add trivy config path option

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ GitHub Actions for [trivy config](https://aquasecurity.github.io/trivy/latest/do
 
 ![image](https://github.com/suzuki-shunsuke/trivy-config-action/assets/13323303/e4d7f6f7-3df3-44bb-8f98-535173ce096e)
 
---
+---
```
Comment (suzuki-shunsuke) [inline README.md:7]:
```suggestion
--
```

This is intentional. Please don't fix.

**Comment by suzuki-shunsuke** [comment]
Thank you for your contribution!

File: action.yaml [hunk 2]
```diff
@@ -13,6 +13,10 @@ inputs:
     description: 'Whether a comment is posted with github-comment'
     required: false
     default: 'false'
+  config_path:
+    description: 'Path to trivy config file'
+    required: false
+    default: 'trivy.yaml'
```
Comment (suzuki-shunsuke) [inline action.yaml:19]:
```console
$ trivy config --help
```

```
  -c, --config string             config path (default "trivy.yaml")
```

Looks good.

**Comment by suzuki-shunsuke** [comment]
I added the contribution guide.

- https://github.com/suzuki-shunsuke/trivy-config-action/blob/main/CONTRIBUTING.md
- https://github.com/suzuki-shunsuke/oss-contribution-guide

Please don't do force pushes after opening pull requests.

**Comment by suzuki-shunsuke** [comment]
v0.2.1 is out 🎉
https://github.com/suzuki-shunsuke/trivy-config-action/releases/tag/v0.2.1

**Comment by kynefuk** [comment]
Thanks for the quick release 🙏

**Pull request merged**